app/src/main/kotlin/com/mapzen/erasermap/view/InstructionAdapter.kt

```diff
@@ -11,8 +11,8 @@ import com.mapzen.erasermap.util.DisplayHelper
 import com.mapzen.valhalla.Instruction
 import java.util.ArrayList
 
-public class InstructionAdapter(val context: Context, val instructions: ArrayList<Instruction>,
-        val pager: RouteModeView) : PagerAdapter() {
+public class InstructionAdapter(private val context: Context,
```

Comment:
Kotlin automatically converts constructor arguments to properties if you put `val` or `var` in front and default scope is package private. Just wanted to update these so they are not visible outside the class.